### PR TITLE
fix: move service refresh list

### PR DESCRIPTION
**Pull request by choykarl**
Fix https://github.com/tisfeng/Easydict/commit/dced9d27b49875b6cf6ac386fee3526306b7a906

### CODE REVIEW

File: Easydict/Swift/View/SettingView/Tabs/TabView/ServiceTab.swift [hunk 1]
```diff
@@ -77,7 +77,7 @@ private class ServiceTabViewModel: ObservableObject {
 
     @Published var selectedService: QueryService?
 
-    private(set) var services: [QueryService]
+    @Published private(set) var services: [QueryService]
```
Comment (tisfeng) [inline Easydict/Swift/View/SettingView/Tabs/TabView/ServiceTab.swift:80]: `services` is private, it's not used outside, what's the point of changing it to `@Published`?

And, what problem will the old code cause?

**Review by tisfeng — CHANGES_REQUESTED** [review]

File: Easydict/Swift/View/SettingView/Tabs/TabView/ServiceTab.swift (hunk 1 above)
Comment (choykarl) [reply Easydict/Swift/View/SettingView/Tabs/TabView/ServiceTab.swift:80]: 现在切换服务顺序不会刷新了

Comment (tisfeng) [reply Easydict/Swift/View/SettingView/Tabs/TabView/ServiceTab.swift:80]: 哦，明白了，我的问题，之前修改 dced9d2 把这里的 `@Published` 弄丢了 😥

**Comment by tisfeng** [comment]
这问题有点烦，过两天发个修复版本吧。

**Review by tisfeng — APPROVED** [review]

**Comment by choykarl** [comment]
我觉得可以，确实有点影响体验

**Pull request merged**